exercises/concept/windowing-system/.docs/instructions.md

```diff
@@ -0,0 +1,159 @@
+# Instructions
+
+In this exercise, you will be simulating a windowing based computer system.
+You will create some windows that can be moved and resized.
+The following image is representative of the values you will be working with below.
+
+```
+                  <--------------------- screenSize.width --------------------->
+
+       ^          ╔════════════════════════════════════════════════════════════╗
+       |          ║                                                            ║
+       |          ║          position.x,_                                      ║
+       |          ║          position.y  \                                     ║
+       |          ║                       \<----- size.width ----->            ║
+       |          ║                 ^      *──────────────────────┐            ║
+       |          ║                 |      │        title         │            ║
+       |          ║                 |      ├──────────────────────┤            ║
+screenSize.height ║                 |      │                      │            ║
+       |          ║            size.height │                      │            ║
+       |          ║                 |      │       contents       │            ║
+       |          ║                 |      │                      │            ║
+       |          ║                 |      │                      │            ║
+       |          ║                 v      └──────────────────────┘            ║
+       |          ║                                                            ║
+       |          ║                                                            ║
+       v          ╚════════════════════════════════════════════════════════════╝
+```
+
+## 1. Define a `size` struct for storing the dimensions of the window
+
+Define a `struct` named `size`.
+It should have two member variables `width` and `height` that store the window's current dimensions.
+The constructor function should accept initial values for these fields.
```

Comment:
here and in task 2: "constructor function" sounds rather uncommon. How about just "constructor"?

These instructions use the terms "member variables" and "fields" as synonyms. Maybe we should stick to one term.